rtc/rtc-supported-functionality.md

```diff
@@ -0,0 +1,81 @@
+
+The Real-Time Collaboration (RTC) plugin supports a subset of the features that {{site.productname}} provides.
+
+> **Warning**: Enabling plugins not shown here is not supported and may lead to unexpected behavior.
+
+The following plugins are currently supported:
+
+* [Advanced List (`advlist`)]({{site.baseurl}}/plugins/opensource/advlist/)
+* [Autoresize (`autoresize`)]({{site.baseurl}}/plugins/opensource/autoresize/)
+* [Character Map (`charmap`)]({{site.baseurl}}/plugins/opensource/charmap/)
+* [Emoticons (`emoticons`)]({{site.baseurl}}/plugins/opensource/emoticons/)
+* [Help (`help`)]({{site.baseurl}}/plugins/opensource/help/)
+* [Horizontal Rule (`hr`)]({{site.baseurl}}/plugins/opensource/hr/)
+* [Image (`image`)]({{site.baseurl}}/plugins/opensource/image/)
+  * Captioned images are not yet supported.
+  * The [`image_caption`]({{site.baseurl}}/plugins/opensource/image/#image_caption) option is not supported.
+* [Insert Date/Time (`insertdatetime`)]({{site.baseurl}}/plugins/opensource/insertdatetime/)
+* [Link (`link`)]({{site.baseurl}}/plugins/opensource/link/)
+  * The [`rel_list`]({{site.baseurl}}/plugins/opensource/link/#rel_list) option is not supported.
+* [Lists (`lists`)]({{site.baseurl}}/plugins/opensource/lists/)
+* [MoxieManager (`moxiemanager`)]({{site.baseurl}}/plugins/premium/moxiemanager/)
+* [PowerPaste (`powerpaste`)]({{site.baseurl}}/plugins/premium/powerpaste/)
+* [Print (`print`)]({{site.baseurl}}/plugins/opensource/print/)
+* [Save (`save`)]({{site.baseurl}}/plugins/opensource/save/)
+* [Tab Focus (`tabfocus`)]({{site.baseurl}}/plugins/opensource/tabfocus/)
+* [Tiny Drive (`tinydrive`)]({{site.baseurl}}/plugins/premium/tinydrive/)
+* [Visual Blocks (`visualblocks`)]({{site.baseurl}}/plugins/opensource/visualblocks/)
+* [Word Count (`wordcount`)]({{site.baseurl}}/plugins/opensource/wordcount/)
+
+## Integration considerations
+
+When planning to integrate Real-Time Collaboration into your application or website, consider the following details and how they impact your use-case. If you are in doubt, don't hesitate to [contact us]({{site.contactpage}}) and discuss your requirements.
+
+* Do not load content into {{site.productname}} that was created using a plugin not yet supported in RTC, such as: Tables, Media embed, Checklist, or Table of contents. Without the plugin enabled to protect the relevant content, you risk making changes to the HTML in a way it's not longer recognized by the plugin.
+* If content is created in an RTC session, then updated outside of an _RTC-enable_ {{site.productname}} editor, a new document ID (`documentId`) is needed for subsequent RTC sessions. The RTC plugin does not currently provide APIs for updating the content programmatically. If your use-case requires you to update the content outside of the RTC clients, your server will need to generate a new document ID and distribute it to all collaborators and block or otherwise prevent editing or collaborating with the old document ID.
```

Comment:
```suggestion
* If content is created in an RTC session, then updated outside of an _RTC-enabled_ {{site.productname}} editor, a new document ID (`documentId`) is needed for subsequent RTC sessions. The RTC plugin does not currently provide APIs for updating the content programmatically. If your use-case requires you to update the content outside of the RTC clients, your server will need to generate a new document ID and distribute it to all collaborators and block or otherwise prevent editing or collaborating with the old document ID.
```